docs/integrations/sources/google-adwords.md

```diff
@@ -0,0 +1,43 @@
+# Google Adwords
+
+## Overview
+
+The Adwords source supports Full Refresh syncs. That is, every time a sync is run, Airbyte will copy all rows in the tables and columns you set up for replication into the destination in a new table.
+
+This Adwords source wraps the [Singer Adwords Tap](https://github.com/singer-io/tap-adwords).
+
+### Output schema
+
+Several output streams are available from this source \(accounts, campaigns, ads, etc.\) For a comprehensive output schema [look at the Singer tap schema files](https://github.com/singer-io/tap-adwords/tree/master/tap_adwords/schemas).
+
+### Features
+
+| Feature | Supported? |
+| :--- | :--- |
+| Full Refresh Sync | Yes |
+| Incremental Sync | No |
+| Replicate Incremental Deletes | No |
+| SSL connection | Yes |
+
+### Performance considerations
+
+This source is constrained by whatever API limits are set for the Google Adwords Manager that is used. You can read more about those limits in the [google developer docs](https://developers.google.com/adwords/api/faq#access).
+
+## Getting started
+
+### Requirements
+
+* Google Adwords Manager Account with an approved Developer Token (note: In order to get API access to Google Adwords, you must have a "manager" account. This must be created separately from your standard account. You can find more information about this distinction in the [google ads docs](https://ads.google.com/home/tools/manager-accounts/).)
+
+### Setup guide
```

Comment:
this process is too hard. even more argument to move forward with supporting oauth properly. but for now it is identical to what you have to do with the singer tap.

Reply:
Agreed. same feeling about FB setup.